src/modules/referrals/referralUtil.ts

```diff
@@ -0,0 +1,44 @@
+import { EnumFilter } from '../dataFetching/types';
+import { localResolvePickList } from '../form/util/formUtil';
+import { PickListOption, ReferralPostingStatus } from '@/types/gqlTypes';
+
+const sortValue = (status: ReferralPostingStatus) => {
+  switch (status) {
+    case ReferralPostingStatus.AssignedStatus:
+      return 1;
+    case ReferralPostingStatus.AcceptedPendingStatus:
+      return 2;
+    case ReferralPostingStatus.AcceptedStatus:
+      return 3;
+    case ReferralPostingStatus.DeniedPendingStatus:
+      return 4;
+    case ReferralPostingStatus.DeniedStatus:
+      return 5;
+    default:
+      return 10;
+  }
+};
+
+// Define a filter for ReferralPostingStatuses
+export const getReferralFilter = (
```

Comment:
This is fine. Maybe could check how often the caller re-renders, maybe triggering table render. If so perhaps should memoize, something like:

```ts
export const useGetReferralFilter = (statuses) => {
  return useMemo( ()=> getReferralFilter(statuses), [statuses] )
}
```